src/pages/BlogPostPage.tsx

```diff
@@ -192,7 +192,7 @@ const BlogPostPage: React.VFC = () => {
             </div>
             <div className="mb-5">
               {post?.tags.map(tag => (
-                <Link key={tag} to={`/posts/?tags=${tag}`} className="mr-2">
+                <Link key={tag} to={`/posts/?categories=${tag}`} className="mr-2">
```

Comment:
Do not need to change this one. we should support two kinds of query params: tags and categories
not change all the tags to categories